app/views/sales/_invoice.pdf.haml

```diff
@@ -1,5 +1,8 @@
 - customer = sale.customer.decorate
 - organization = sale.organization.decorate
+- comment = ''
+- if sale.comments.first
+  - comment = sale.comments.first.body
```

Comment:
You have a point of invoice comment as a database field. The difference between text on invoice and internal comments becomes clearer to the user. The text is set when the sale-head is registered. 

Comments are saved in one place but return to the parent edit function. It matters in issue 276.